### ISSUE TITLE
Why not migrate to .Net6.0?

### ISSUE THREAD

**Issue by Ghislain1** [ArcadeRenegade/SidebarDiagnostics#363]
**1. Is this a feature request?**

*If yes then sorry but this project is not taking feature requests at this time.*

**2. Your Operating System**

**3. App Version**

**4. Any Relevant Settings**

**5. Steps to Reproduce**

**Comment by ArcadeRenegade** [comment]
I haven't had much time. Is there a significant performance benefit?

**Comment by Ghislain1** [comment]
Yes ! you can read release note .Net5.0.  .Net Framework is almost dead 👎

**Comment by ArcadeRenegade** [comment]
I'll be porting it over at some point this year. I need to check if net 5 is installed by default on most windows 10 computers now. I had issues with using the latest version of net framework because people didn't have it on their pcs.

**Comment by Ghislain1** [comment]
https://github.com/ArcadeRenegade/SidebarDiagnostics/pull/271
starts doing it  at all !!
